Mathlib/LinearAlgebra/QuadraticForm/Basic.lean

```diff
@@ -379,6 +380,35 @@ theorem choose_exists_companion : Q.exists_companion.choose = polarBilin Q :=
       add_sub_cancel_left]
 #align quadratic_form.some_exists_companion QuadraticMap.choose_exists_companion
 
+protected theorem map_sum {ι} [DecidableEq ι] (Q : QuadraticMap R M N) (s : Finset ι) (f : ι → M) :
+    Q (∑ i ∈ s, f i) = ∑ i ∈ s, Q (f i) +
+      ∑ ij in s.sym2.filter (¬ ·.IsDiag),
+        Sym2.lift ⟨fun i j => polar Q (f i) (f j), fun _ _ => polar_comm _ _ _⟩ ij := by
```

Comment:
nit
```suggestion
        Sym2.lift ⟨fun i j => polar Q (f i) (f j), fun _ _ => polar_comm ..⟩ ij := by
```

Reply:
I don't really find this an improvement, it makes more work for the reader and the elaborator